iib/web/api_v1.py

```diff
@@ -99,6 +99,7 @@ def _get_rm_args(
         request.distribution_scope,
         flask.current_app.config['IIB_BINARY_IMAGE_CONFIG'],
         payload.get('build_tags', []),
+        flask.current_app.config['IIB_INDEX_TO_GITLAB_PUSH_MAP'],
```

Comment:
We probably also need to add this to `handle_fbc_operations`

Reply:
I've added it to the `fbc_operations`, PTAL again